app/webpacker/src/stylesheets/_feature-panel.scss

```diff
@@ -9,4 +9,8 @@ $feature-panel-border-colour: #333;
   &.last-feature-panel {
     border: 0;
   }
+
+  &--shaded {
+    background-color: #f0f4f5;
```

Comment:
Should we explicitly name this color?

Reply:
tbh - we shouldn't be using a colour outside the official palette - it adds nothing for accessibility. Following the designs though